easybuild/toolchains/linalg/intelmkl.py

```diff
@@ -84,6 +85,7 @@ def _set_blas_variables(self):
         interfacemap = {
             TC_CONSTANT_INTELCOMP: 'intel',
             TC_CONSTANT_GCC: 'gf',
+            TC_CONSTANT_PGI: 'intel',
```

Comment:
from the output of `eb --from-pr 2900 HPL-2.1-pomkl-2016.03.eb -x`:

```
  export BLAS_MT_STATIC_LIBS="libmkl_intel_lp64.a,libmkl_pgi_thread.a,libmkl_core.a"
  export BLAS_STATIC_LIBS="libmkl_intel_lp64.a,libmkl_sequential.a,libmkl_core.a"
```

the `libmkl_intel_lp64.a` should be `libmkl_pgi_lp64.a` instead?

However, there's no `libmkl_pgi_lp64.a` available...